### DISCUSSION TITLE
ディスカッションのテスト

### DISCUSSION THREAD

**Discussion by nuta** [nuta/microkernel-book#1] [General]
こんにちは

**Comment by nuta** [comment]
[![https://xkcd.com/1597/](https://imgs.xkcd.com/comics/git.png)](https://xkcd.com/1597/)

**Comment by ohfujimasa73** [comment]
make を実行すると下記エラーが出るのですが、どう対処したらいいのでしょうか

error: unable to create target: 'No available targets are compatible with triple "riscv32"'
1 error generated.
make: *** [build/kernel/main.o] エラー 1

**Reply by nuta** [reply]
@ohfujimasa73 どうやら想定していない構成のコンパイラ (clang) が利用されているようです。お手数ですが、[こちらのフォーム](https://github.com/nuta/microkernel-book/discussions/new?category=help) から新しいスレッドを作成していただけますでしょうか 🙏

**Reply by ohfujimasa73** [reply]
@nuta さん
新しいスレッドを作成しました。
Ubuntuでは問題なく実施できています。
